advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+
+15. Provide required information and click Ok on Copy Request Properties Screen.
+
+![Copy Request](Images/copyrequest.png)
+
+## Save the Model
+
+The Model created in the above steps needs to be saved, validated and committed before the Replication can be performed.
+
+1. To save the Model, click File and select Save option and provide the name for the Model.
+
+2. Right Click Enterprise Data Movement Model and select Validate to validate the Model.
+
+3. No error will be displayed if the Model is valid.
+
+![Model Validation](Images/validmodel.png)
+
+4. Right Click Enterprise Data Movement Model and select  Commit —> Full.
+
+![Model Commit](Images/modelcommit.png)
+
+5. Click Ok on Commit Model Screen. This will create the Model files.
+
+## Execute Copy Replication using Connect CDC MonCon
+
+Connect CDC MonCon is a GUI application, separate from the Connect CDC Director, that you use for monitoring and control functions. We will use Connect CDC MonCon to execute the Copy Replication. Copy Replication will copy all the data from the Source Tables to the Target Tables.
+
+1. Open Connect CDC MonCon.
+
+2. Click Model —> New Model from Menu.
+
+3. Provide Hostname/IP and select the saved Model from the drop down list.
+
+![New Model](Images/newmodel.png)
+
+4. In the Request Section, select the Distribution Model that was created for copying data created in the above steps and right click and select start.
+
+![Distribution Model](Images/distributionmodel.png)
+
+5. Click on Process to see the progress.
+
+![Process](Images/Process.png)
+
+6. Open pgAdmin once the copy operation is completed successfully and connect the EDB Postgres Advanced Server Target Instance and check the tables and the data is copied to the tables.
+
+![pgAdmin](Images/pgAdmincopydone.png)
+
+![pgAdmin](Images/pgAdmincopydone2.png)
+
+## Create Replication Distribution using Connect CDC Director
+
+Once the Copy replication operation is successful then a Replication option will be executed which will replicate all the changes made on the Source Tables to the Target Tables.
+
+1. Open Connect CDC Director.
+
+2. Create the Replication Request from Distributions —> Newly Created Distribution —> Requests —> New Request —> Replication.
+
+3. Enter the name and press Ok.
+
+![Replication Request](Images/replicationrequest.png)
+
+![Replication Request](Images/replicationrequest2.png)
+
+4. Save the changes in the Model.
+
+5. Right Click Enterprise Data Movement Model and select Validate to validate the Model.
+
+6. Right Click Enterprise Data Movement Model and select  Commit —> Full.
+
+7. Click Ok on Commit Model Screen. This will create the Model files.
+
+## Execute Replication option using Connect CDC MonCon
+
+Replication distribution will replicate all the changes done on the Source Tables to the Target Tables.
```

Comment:
Should we mention here that this is just an option? Like if they want to use MonCon? It just slightly reads that they need to do both. Unless that is the case

Reply:
Removed "option".